### ISSUE TITLE
ng-repeat directive doesn't work

### ISSUE THREAD

**Issue by nick-merrill** [softwaredoug/inspector-gadget#1]
Thanks for a great module!

Unfortunately, I can't get a simple `ng-repeat` directive to work inside the `<inspector-content>` tag.

For example, this

```
<inspector-gadget>
    <inspector-content>
        <span ng-repeat="i in [1, 2, 3]">[[ i ]]</span>
    </inspector-content>
</inspector-gadget>
```

results in no content output.

Any thoughts?  Thanks!

**Comment by softwaredoug** [comment]
Awesome find.

In the transclude function I presumed that I had access to the original HTML of inspector-title and inspector-content. Then I later compile this when giving it off to bootstrap's popover.

Not true, angular precompiles this before you get to act to do the transclusion and there's no straight-forward way to get the original markup before compilation happened. The child nodes are compiled before pre/postlink functions.

[ngNonBindable](https://docs.angularjs.org/api/ng/directive/ngNonBindable) appears to be a temporary work around. For now, I would apply this to inspector-content/title tags as follows:

```
<inspector-gadget>
    <inspector-content ng-non-bindable>
        <span ng-repeat="i in [1, 2, 3]">[[ i ]]</span>
    </inspector-content>
</inspector-gadget>
```

I may have to require this if I can't find away to force my directive be non-bindable

**Comment by softwaredoug** [comment]
[ngNonBindable](https://github.com/angular/angular.js/blob/g3_v1_2/src/ng/directive/ngNonBindable.js) achieves this magic through the terminal directive option. More on [terminal](http://stackoverflow.com/questions/15266840/how-to-understand-the-terminal-of-directive)

**Comment by softwaredoug** [comment]
Fixed in cbfd14e4d01164ce0c8464c51d30ca6e56df41f4 (v0.1.3)

Also see second [example](http://softwaredoug.github.io/inspector-gadget/#/home) that uses ng-repeat.

Should be fixed for you. But certainly let me know if you have any problems. Thanks again!

**Comment by nick-merrill** [comment]
Belated thanks @softwaredoug!  I definitely would not have figured that one out.
